src/m_menu.c

```diff
@@ -3320,8 +3307,6 @@ setup_menu_t stat_settings1[] =  // Status Bar and HUD Settings screen
 
   {"WIDGET APPEARANCE",S_SKIP|S_TITLE,m_null,M_X,M_Y+stat1_title3*M_SPC},
 
-  {"BACKPACK SHIFTS AMMO COLORS"    ,S_YESNO,m_null,M_X,M_Y+stat1_backpack*M_SPC,  {"hud_backpack_thresholds"}},
```

Comment:
Fixed here: https://github.com/fabiangreffrath/woof/pull/1303/commits/bf12f5c48a7e48ce04782eb900c97944507d8dd2

Reply:
Sorry, I was mistaken. Please disable `hud_backpack_thresholds` by default.

Reply:
Okay, fixed [here](https://github.com/fabiangreffrath/woof/compare/40c062a5e276cba94b540e2f7567e8cd359d3f6f..66245cbd66bf9b4216358e1ec0a8a665f1dbbda0).

Reply:
Thank you. But still, my problem with removing these two features from the menu is that they are very subjective. You either get used to one behavior or the other. And thus I can imagine that quite some people do indeed change them. 

Maybe they should get moved up a bit beneath the "Status Bar" subjective line and "Gray Percent" gets removed instead. And maybe they should be grayed out if the standard colors are enabled for the status bar numbers. 

This, and the fact that removing these items doesn't really help much. It doesn't save us a menu page or remove a wall of text. These are small self-descriptive menu items on a well structured page IMHO.

Reply:
Okay everything for both those settings reverted [here](https://github.com/fabiangreffrath/woof/compare/66245cbd66bf9b4216358e1ec0a8a665f1dbbda0..4e8693d63a8b36aad58659758bdee4a5fa8756af). This PR is just for the ones that seem obvious and easy to remove, so I'm intentionally avoiding any kind of restructuring or overhaul type changes.

Reply:
> This, and the fact that removing these items doesn't really help much. It doesn't save us a menu page or remove a wall of text. These are small self-descriptive menu items on a well structured page IMHO.

How about getting rid of the Boom menu structure? I like trying to consolidate all the UI options on one page, so we don't need separate HUD and Automap pages.

> And maybe they should be grayed out if the standard colors are enabled for the status bar numbers.

Please don't change the vanilla colors. It always really annoyed me when Doom ports changed the appearance of the status bar, I consider it iconic.

Reply:
> Please don't change the vanilla colors. It always really annoyed me when Doom ports changed the appearance of the status bar, I consider it iconic.

This is not my intention. I was suggesting to gray out menu options that control how status bar colors change if status bar color changing is disabled in the first place. 😉

Reply:
He, why didn't I notice this before? "General" should be the first sub-menu of "Setup", yes of course!

Reply:
> Also, I think with the latest changes our menu isn't that bad anymore, hm?

I think there's still a considerable amount of simplifying and restructuring to be done. I didn't know about it at the time, but nearly everything I originally removed in this PR was also done in DietBoom. I think the readme.md for that project makes some good points about how we should be thinking about every setting. I don't agree with every change, but I agree with the overall philosophy.

Reply:
> But @JNechaevsky told me that this is a very bad idea.

Yes, please, from my _artist's_ POV which is far better than my _programmer's_ POV, I'm really asking you to avoid overscaling of `STCFN` as a Black Plague... HSV coloring is doing it's awesome job for visual distinguishing, and if some extra neat accent is really needed, you can always draw `TRANMAP`'s shadow below text.